ghost/api-framework/lib/headers.js

```diff
@@ -122,16 +122,26 @@ module.exports = {
         }
 
         const locationHeaderDisabled = apiConfigHeaders?.location === false;
-        const hasFrameData = frame?.method === 'add' && result[frame.docName]?.[0]?.id;
+        const hasFrameData = ['add', 'copy'].includes(frame?.method) && result[frame.docName]?.[0]?.id;
```

Comment:
I think a cleaner way to pass this information would be allowing for a custom option in the endpoint definition itself. Checking for method names in the framework gives a code smell tbh. 🤔 

We could introduce something like `frame.location` and determine the returned header base on the value there, this way we would not create additional coupling between framework and method names inside of endpoint definitions.

Reply:
We don't have to go throught all controllers, just check if the new config value is present than use it and if not leave the previous logic as is.

Reply:
See what you think to https://github.com/TryGhost/Ghost/pull/16767/commits/15191f86ba1e1b1c9c670505c6b3c96ebe3db51e 👍